src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/ReplicationProtectedItem/NewAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -26,64 +26,75 @@ namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
     [Cmdlet(
         VerbsCommon.New,
         "AzureRmRecoveryServicesAsrReplicationProtectedItem",
-        DefaultParameterSetName = ASRParameterSets.DisableDR,
+        DefaultParameterSetName = ASRParameterSets.EnterpriseToEnterprise,
         SupportsShouldProcess = true)]
     [Alias("New-ASRReplicationProtectedItem")]
     [OutputType(typeof(ASRJob))]
     public class NewAzureRmRecoveryServicesAsrReplicationProtectedItem : SiteRecoveryCmdletBase
     {
-        private Job jobResponse;
-
         /// <summary>
-        ///     Job response.
+        ///    Switch Paramter to create VMwareToAzure / InMageAzureV2 policy.
         /// </summary>
-        private PSSiteRecoveryLongRunningOperation response;
-
+        [Parameter(
+            Position = 0,
+            ParameterSetName = ASRParameterSets.VMwareToAzure,
+            Mandatory = true)]
+        public SwitchParameter VMwareToAzure { get; set; }
+        
         /// <summary>
-        ///     Gets or sets Replication Protected Item.
+        ///    Switch Paramter to create HyperVSiteToAzure policy.
         /// </summary>
         [Parameter(
-            ParameterSetName = ASRParameterSets.EnterpriseToEnterprise,
-            Mandatory = true,
-            ValueFromPipeline = true)]
-        [Parameter(
+            Position = 0,
             ParameterSetName = ASRParameterSets.EnterpriseToAzure,
-            Mandatory = true,
-            ValueFromPipeline = true)]
+            Mandatory = false)]
         [Parameter(
+            Position = 0,
             ParameterSetName = ASRParameterSets.HyperVSiteToAzure,
-            Mandatory = true,
-            ValueFromPipeline = true)]
+            Mandatory = false)]
+        public SwitchParameter HyperVToAzure { get; set; }
+
+        /// <summary>
+        ///    Switch Paramter to create HyperVSiteToHyperVSite policy.
+        /// </summary>
+        [Parameter(
+            Position = 0,
+            ParameterSetName = ASRParameterSets.EnterpriseToEnterprise,
+            Mandatory = false)]
+        public SwitchParameter VmmToVmm { get; set; }
+
+        /// <summary>
+        ///     Gets or sets Replication Protected Item.
+        /// </summary>
+        [Parameter(Mandatory = true)]
```

Comment:
@viverm this parameter no longer accepts its value from the pipeline, which is a breaking change

Reply:
@cormacpayne      

bug . fixing it.